MultiVersions/Forge/main/integrateddynamics/lang/zh_tw.json

```
@@ -0,0 +1,1873 @@
+    "aspect.integrateddynamics.write.double.effect.particle.minecraft_electric_spark.info": "以特定速率釋放銅方塊還原粒子",
+    "aspect.integrateddynamics.write.double.effect.particle.minecraft_scrape": "粒子效果：銅方塊除鏽",
+    "aspect.integrateddynamics.write.double.effect.particle.minecraft_scrape.info": "以特定速率釋放銅方塊除鏽粒子",
+
+    "_comment": "# Block aspects",
+    "aspect.integrateddynamics.read.boolean.block.block": "是否為方塊",
+    "aspect.integrateddynamics.read.boolean.block.block.info": "檢查偵測地點是否存在方塊",
+
+    "aspect.integrateddynamics.read.string.block.dimension": "維度",
+    "aspect.integrateddynamics.read.string.block.dimension.info": "方塊所在維度的 ID",
+    "aspect.integrateddynamics.read.integer.block.posx": "X 座標",
+    "aspect.integrateddynamics.read.integer.block.posx.info": "方塊所在的 X 座標",
+    "aspect.integrateddynamics.read.integer.block.posy": "Y 座標",
+    "aspect.integrateddynamics.read.integer.block.posy.info": "方塊所在的 Y 座標",
+    "aspect.integrateddynamics.read.integer.block.posz": "Z 座標",
+    "aspect.integrateddynamics.read.integer.block.posz.info": "方塊所在的 Z 座標",
+    "aspect.integrateddynamics.read.nbt.block.tile": "方塊實體 NBT",
+    "aspect.integrateddynamics.read.nbt.block.tile.info": "方塊實體的 NBT",
+    "aspect.integrateddynamics.read.string.block.biome": "生態域",
+    "aspect.integrateddynamics.read.string.block.biome.info": "方塊所在生態域的名字",
+    "aspect.integrateddynamics.read.integer.block.light": "亮度等級",
+    "aspect.integrateddynamics.read.integer.block.light.info": "方塊的亮度等級",
+
+    "aspect.integrateddynamics.read.block.block": "方塊",
+    "aspect.integrateddynamics.read.block.block.info": "被偵測的方塊",
+
+    "_comment": "# Entity aspects",
+    "aspect.integrateddynamics.read.integer.entity.itemframerotation": "物品展示框旋轉",
+    "aspect.integrateddynamics.read.integer.entity.itemframerotation.info": "物品在物品展示框旋轉的狀況",
+
+    "aspect.integrateddynamics.read.list.entity.entities": "實體",
+    "aspect.integrateddynamics.read.list.entity.entities.info": "在特定範圍裡的實體",
+    "aspect.integrateddynamics.read.list.entity.players": "玩家",
+    "aspect.integrateddynamics.read.list.entity.players.info": "在世界上的玩家",
+
+    "aspect.integrateddynamics.read.entity.entity": "實體",
+    "aspect.integrateddynamics.read.entity.entity.info": "在特定範圍裡的實體",
+
+    "aspect.integrateddynamics.read.itemstack.entity.itemframecontents": "物品展示框",
+    "aspect.integrateddynamics.read.itemstack.entity.itemframecontents.info": "在物品展示框的物品",
+
+    "_comment": "# Extra-Dimensional aspects",
+    "aspect.integrateddynamics.read.integer.extradimensional.random": "隨機數",
+    "aspect.integrateddynamics.read.integer.extradimensional.random.info": "隨機整數值",
+    "aspect.integrateddynamics.read.integer.extradimensional.playercount": "玩家數量",
+    "aspect.integrateddynamics.read.integer.extradimensional.playercount.info": "伺服器內的玩家數量",
+    "aspect.integrateddynamics.read.integer.extradimensional.ticktime": "每刻時間",
```

參考**frametime**翻譯，加入「此數值為計算每一刻所耗費之時間」的概念
```suggestion
    "aspect.integrateddynamics.read.integer.extradimensional.ticktime": "每刻耗時",
```